src/App.tsx

```diff
@@ -101,13 +115,16 @@ const receiptStateToDisplayableResult = (
   }
 };
 
-export interface L1ToL2MessageStatusDisplay {
+export interface MessageStatusDisplay {
+  isRetryables: boolean;
+  [x: string]: any;
   text: string;
   alertLevel: AlertLevel;
   showRedeemButton: boolean;
   explorerUrl: string;
   l2Network: L2Network;
   l1ToL2Message: IL1ToL2MessageReader;
+  ethDepositMessage: EthDepositMessage; 
   l2TxHash: string;
 }
```

Comment:
```suggestion
interface MessageStatusDisplayBase{
  text: string;
  alertLevel: AlertLevel;
  showRedeemButton: boolean;
  explorerUrl: string;
  l2Network: L2Network;
  l2TxHash: string;
}
interface MessageStatusDisplayRetryable extends MessageStatusDisplayBase{
  l1ToL2Message: IL1ToL2MessageReader;
  ethDepositMessage: undefined;
}
interface MessageStatusDisplayDeposit extends MessageStatusDisplayBase{
  l1ToL2Message: undefined;
  ethDepositMessage: EthDepositMessage; 
}

export type MessageStatusDisplay = MessageStatusDisplayRetryable | MessageStatusDisplayDeposit 
```

Also dropped the isRetryables boolean since we can detect using e.g. `if(msg.l1ToL2Message !== undefined)`